index.js

```diff
@@ -211,7 +213,7 @@ NYC.prototype.instrumentAllFiles = function (input, output, cb) {
     }
 
     if (transform) {
-      code = transform(code, {filename: filename, relFile: inFile})
+      code = transform(code, { filename: filename, relFile: inFile })
```

Comment:
We should add the prettier config to this repo like the istanbuljs monorepo, not blocking this merge, but just noting

Reply:
I'll defer this, I'd like to get a new version published to npm.

FWIW I do not like prettier or standard.  The formatting rules hurt my brain, especially the no semi-colons.